staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3278,6 +3278,12 @@ const (
 	// PodReadyToStartContainers pod sandbox is successfully configured and
 	// the pod is ready to launch containers.
 	PodReadyToStartContainers PodConditionType = "PodReadyToStartContainers"
+	// PodResizePending indicates that the pod has been resized, but kubelet has not
+	// yet allocated the resources.
+	PodResizePending PodConditionType = "PodResizePending"
```

Comment:
Should these say something about what it means if both of these are set?

Reply:
Good idea, thanks! Added to the descriptions